tests/installation/resolve_dependency_issues.pm

```diff
@@ -34,7 +34,13 @@ sub run {
 sub post_fail_hook {
     my $self = shift;
     select_console 'root-console';
-    $self->upload_solvertestcase_logs();
+    if (check_var('UPGRADE', '1')) {
```

Comment:
Maybe we can check error of zypper command feedback instead of check UPGRADE?

Reply:
the yast2 installation failed to wait for manual intervention  https://openqa.nue.suse.com/tests/7919448#step/resolve_dependency_issues/29
and the solvertest case failed for 'zypper not found'
https://openqa.nue.suse.com/tests/7919448#step/resolve_dependency_issues/37